gadget/install/install.go

```diff
@@ -281,6 +293,9 @@ func createPartitions(model gadget.Model, gadgetRoot, kernelRoot, bootDevice str
 	if err != nil {
 		return "", nil, nil, 0, fmt.Errorf("cannot create the partitions: %v", err)
 	}
+	// Tweak laid out data depending on encryption type
+	// TODO: try to build LaidOutVolume with all info in the future
+	setCryptoInfoInLaidOuts(options.EncryptionType, created)
```

Comment:
why do we need to do it again?

Reply:
aren't the encrypted ones always among the created atm?

Reply:
Right, the problem is that all fields in ondisk were being overwritten inside `createMissingPartitions()`, I've fixed that now.